packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offer-details/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "dedicatedCloud_offer_upgrade_custom_report": "Rapport de sauvegarde personnalisable",
+  "dedicatedCloud_offer_upgrade_encrypted_backup": "Sauvegarde chiffrée",
+  "dedicatedCloud_offer_upgrade_retention_points": "{{retention}} points de rétention",
```

What is there is only one ? :)

```suggestion
  "dedicatedCloud_offer_upgrade_retention_point_1": "1 point de rétention",
  "dedicatedCloud_offer_upgrade_retention_point_+": "{{retention}} points de rétention",
```